### PR TITLE
[4.0] fix when error on massmail

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #26680 .

### Summary of Changes
fixed call on boolean

### Testing Instructions
Login to Joomla Admin with admin accounts.

Go to Admin => Mass Mail Users

Enter Subject with a space at the end e.g: "Test Subject "

Enter at Send mail button

or disable sendmail
### Actual result

`Call to member function getError() on bool`

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 5596c66f92ea602e7c4b87426db8211e29b2a3ae<br /><br />Not sure if it's the best way to fix it but it fixes it.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26742">issues.joomla.org/tracker/joomla-cms/26742</a>.</sub>

File: administrator/components/com_users/Model/MailModel.php [hunk 1]
```diff
@@ -218,7 +218,7 @@ public function send()
 		if ($rs !== true)
 		{
 			$app->setUserState('com_users.display.mail.data', $data);
-			$this->setError($rs->getError());
+			$this->setError(Text::_('COM_USERS_MAIL_THE_MAIL_COULD_NOT_BE_SENT'));
```
Comment (SharkyKZ) [inline administrator/components/com_users/Model/MailModel.php]: Haven't tested but Privacy component uses `$mailer->ErrorInfo`.

Comment (alikon) [reply administrator/components/com_users/Model/MailModel.php]: Yes i've sawed that... but in that way it is untranslatable

**Comment by anibalsanchez** [comment]
I have tested this item :white_check_mark: successfully on 5596c66f92ea602e7c4b87426db8211e29b2a3ae<br /><br />Test OK<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26742">issues.joomla.org/tracker/joomla-cms/26742</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26742">issues.joomla.org/tracker/joomla-cms/26742</a>.</sub>

File: administrator/components/com_users/Model/MailModel.php (hunk 1 above)
Comment (SharkyKZ) [reply administrator/components/com_users/Model/MailModel.php]: Can you double check that? I'm almost certain PHPMailer supports translations. And if not, what does this file do? https://github.com/joomla/joomla-cms/blob/5596c66f92ea602e7c4b87426db8211e29b2a3ae/libraries/src/Mail/language/phpmailer.lang-en_gb.php

Comment (wilsonge) [reply administrator/components/com_users/Model/MailModel.php]: PHPMailer definitely should be translatable through that mechanism

Comment (alikon) [reply administrator/components/com_users/Model/MailModel.php]: ok then 
used `$mailer->ErrorInfo`

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**